src/snapshot/prepare_snapshot_received_proposals_airtable.js

```diff
@@ -39,7 +39,7 @@ const prepareProposalsForSnapshot = async (curRound) => {
   // TODO-RA: Proposals are being set to "Accepted" ahead of time, and are not being found.
   // Changed to this for R13 to work => `AND({Round} = "${curRoundNumber}", OR({Proposal State} = "Accepted"), "true")`
   const proposals = await getProposalsSelectQuery(
-    `AND({Round} = "${curRoundNumber}", OR({Proposal State} = "Received", {Proposal State} = "Rejected"), "true")`
+    `AND({Round} = "${curRoundNumber}", {Proposal State} = "Accepted", "true")`
```

Comment:
Are we sure we want to change this? As far as I remember we decided that the initial state of a proposal to be 'Rejected'

Reply:
Only `Accepted` proposals will be included in the voting